assets/javascripts/obs_rsync.js

```diff
@@ -0,0 +1,38 @@
+function postAndRefresh(btn, delay){
+    var post_url = $(btn).attr("post_url");
+    $.ajax({
+        url: post_url,
+        method: 'POST',
+        dataType: 'json',
+        success: function(data) {
+            if (delay) {
+                setTimeout(function() {
+                    location.reload();
```

Comment:
I don't like that we need to reload the page here. I don't require you to change it because it might be some work. But consider that it is perfectly fine (and not that much work) to add routes which return HTML fragments which can be dynamically into the DOM without reloading the whole page.